appscale/tools/admin_api/version.py

```diff
@@ -0,0 +1,327 @@
+""" Represents an Admin API Version resource. """
+
+from __future__ import absolute_import
```

Comment:
I still can't understand why we get ImportError on absolute imports of appscale.tools modules unless we import this future feature.

Reply:
I don't think it's needed in this particular file. When I started working on this branch, this file started out in appscale.tools.

This import has the side effect of disabling the old relative import syntax. In Python 2, you can import a different module in the current module with either `import other_module` or `import .other_module` (newer relative import syntax). Since the namespace package is named "appscale" and there is an "appscale" module in appscale.tools, using `import appscale` from a module in "appscale.tools" is ambiguous. It could either refer to the other module on the same level as the current one or the root "appscale" module.

Reply:
Ohh.. appscale module!
Thanks now I understand=)